components/conversations/CreateConversationForm.js

```diff
@@ -51,21 +83,30 @@ const messages = defineMessages({
 const CreateConversationForm = ({ collectiveId, suggestedTags, onSuccess, disabled, loading }) => {
   const { formatMessage } = useIntl();
   const [createConversation, { error: submitError }] = useMutation(CreateConversationMutation, mutationOptions);
-  const { register, handleSubmit, errors, formState, setValue } = useForm();
+  const [state, dispatch] = useReducer(reducer, initialFormState);
```

Comment:
Here the reducer is ok because we're storing more values. Formik could have helped to make that cleaner.

Reply:
switched to Formik here